.github/workflows/linters.yml

```diff
@@ -0,0 +1,25 @@
+name: Code Linters
```

Comment:
It seems like the previous file ci.yml is already doing only linter activities.
Why don't we move all these steps to that file? We could even rename it from ci.yml to linters.yml to be more accurate.